content/docs/reference/microcontrollers/nano-rp2040.md

```diff
@@ -8,13 +8,13 @@ The [Nano RP2040](https://store.arduino.cc/nano-rp2040-connect) is a tiny develo
 ## Interfaces
 
 | Interface | Hardware Supported | TinyGo Support |
-| --------- | ------------- | ----- |Well, I think 
```

Comment:
Huh, I wonder where that came from. In any case, it's fixed now.

Reply:
Oops?